x-pack/plugins/security_solution/public/timelines/components/timeline/helpers.tsx

```diff
@@ -282,3 +282,14 @@ export const TIMELINE_FILTER_DROP_AREA = 'timeline-filter-drop-area';
 
 export const getNonDropAreaFilters = (filters: Filter[] = []) =>
   filters.filter((f: Filter) => f.meta.controlledBy !== TIMELINE_FILTER_DROP_AREA);
+
+export const isFullScreen = ({
```

Comment:
This is a utility that was part of on of the deleted file but is required at other places as well so moved here.

Reply:
could maybe add a couple of unit tests for it, even though it's super simple?